### PR TITLE
Prepare a 0.3.3 release.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# lang_tester 0.3.3 (2019-10-24)
+
+* Individual tests can now add extra arguments to an invoked command with the
+  `extra-args` field.
+
+* Ensure that, if a command in a chain fails, the whole chain of commands
+  fails. This means that if, for example, compilation of command C fails, we do
+  not try and run C anyway (which can end up doing confusing things like runing
```
Comment (jacob-hughes) [inline CHANGES.md]: typo runing -> running

**Comment by ltratt** [comment]
Can I squash?

**Comment by jacob-hughes** [comment]
Yes please squash!

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**